mission-control/blog/state-based-alerting/index.md

```diff
@@ -0,0 +1,500 @@
+[32m                    [0m[36mfieldPath[0m:[32m metadata.namespace[0m
+[32m                  [0m[36mpath[0m:[32m namespace[0m
+[32m[0m[36mstatus[0m:[36m[0m
+[36m  conditions[0m:
+    -[36m lastProbeTime[0m: null
+      [36mlastTransitionTime[0m:[32m "2025-03-26T10:17:18Z"[0m[36m[0m
+[36m      status[0m:[32m "True"[0m[36m[0m
+[36m      type[0m:[32m PodReadyToStartContainers[0m
+[32m    [0m-[36m lastProbeTime[0m: null
+      [36mlastTransitionTime[0m:[32m "2025-03-26T10:17:16Z"[0m[36m[0m
+[36m      status[0m:[32m "True"[0m[36m[0m
+[36m      type[0m:[32m Initialized[0m
+[32m    [0m-[36m lastProbeTime[0m: null
+      [36mlastTransitionTime[0m:[32m "2025-03-26T10:17:16Z"[0m[36m[0m
+[36m      message[0m:[32m 'containers with unready status: [nginx]'[0m[36m[0m
+[36m      reason[0m:[32m ContainersNotReady[0m
+[32m      [0m[36mstatus[0m:[32m "False"[0m[36m[0m
+[36m      type[0m:[32m Ready[0m
+[32m    [0m-[36m lastProbeTime[0m: null
+      [36mlastTransitionTime[0m:[32m "2025-03-26T10:17:16Z"[0m[36m[0m
+[36m      message[0m:[32m 'containers with unready status: [nginx]'[0m[36m[0m
+[36m      reason[0m:[32m ContainersNotReady[0m
+[32m      [0m[36mstatus[0m:[32m "False"[0m[36m[0m
+[36m      type[0m:[32m ContainersReady[0m
+[32m    [0m-[36m lastProbeTime[0m: null
+      [36mlastTransitionTime[0m:[32m "2025-03-26T10:17:16Z"[0m[36m[0m
+[36m      status[0m:[32m "True"[0m[36m[0m
+[36m      type[0m:[32m PodScheduled[0m
+[32m  [0m[36mcontainerStatuses[0m:
+    -[36m image[0m:[32m nginx:invalid[0m
+[32m      [0m[36mimageID[0m:[32m ""[0m[36m[0m
+[36m      lastState[0m: {}[36m[0m
+[36m      name[0m:[32m nginx[0m
+[32m      [0m[36mready[0m:[95m false[0m
+[95m      [0m[36mrestartCount[0m:[95m 0[0m
+[95m      [0m[36mstarted[0m:[95m false[0m
+[95m      [0m[36mstate[0m:[36m[0m
+[36m        waiting[0m:[36m[0m
+[36m          message[0m:[32m Back-off pulling image "nginx:invalid"[0m
+[32m          [0m[36mreason[0m:[32m ImagePullBackOff[0m
+[32m  [0m[36mhostIP[0m:[32m 10.0.5.138[0m
+[32m  [0m[36mhostIPs[0m:
+    -[36m ip[0m:[32m 10.0.5.138[0m
+[32m  [0m[36mphase[0m:[32m Pending[0m
+[32m  [0m[36mpodIP[0m:[32m 10.0.5.78[0m
+[32m  [0m[36mpodIPs[0m:
+    -[36m ip[0m:[32m 10.0.5.78[0m
+[32m  [0m[36mqosClass[0m:[32m BestEffort[0m
+[32m  [0m[36mstartTime[0m:[32m "2025-03-26T10:17:16Z"[0m
+</TerminalOutput>
+
+
+
+While standards exist for exposing metrics, there's no equivalent standard for exposing the thresholds or conditions that trigger alerts. This leads to fragmentation and complexity in monitoring setups.
+
+[is-healthy](https://github.com/flanksource/is-healthy) is a tool designed to assess and report the health status of Kubernetes and other cloud resources (such as AWS) without the limitations of metric-based approaches.
```

Comment:
Does mentioning is-healthy add too much value here ?

Unless, we want to promote its usage to people as well (in which case we should probably add more things about it, eg. support for flux,argo,crossplane etc. ?